src/environment.js

```diff
@@ -63,9 +63,17 @@ import {
   ThrowIfMightHaveBeenDeleted,
 } from "./methods/index.js";
 import * as t from "babel-types";
+import { TypesDomain, ValuesDomain } from "./domains/index.js";
 
 const sourceMap = require("source-map");
 
+export function deriveGetBinding(realm: Realm, binding: Binding) {
+  let types = new TypesDomain(Value);
```

Comment:
You may as well use TypesDomain.topVal in this case.

Reply:
Ok.